RecoHI/HiTracking/test/testTracking_cfg.py

```diff
@@ -49,7 +48,8 @@
 	),
     heavyIonHLTVertexing = cms.untracked.PSet(
         limit = cms.untracked.int32(-1)
-    )
+    ),
+    MinBiasTracking = cms.untracked.PSet()
```

Comment:
I'm not sure I understand why MinBiasTracking was added here, but not the other categories.

Reply:
The other categories are explicitly mentioned in other _destinations_ in the MessageLogger configuration. That is sufficient to turn them on. The `MinBiasTracking` is not referenced anywhere else so needed to be explicitly added in order to keep the previous behavior.

Reply:
oh, right, I see now. thanks!